Cargo.lock

```diff
@@ -1672,18 +1646,32 @@ dependencies = [
 
 [[package]]
 name = "curve25519-dalek"
-version = "4.0.0-rc.1"
+version = "4.1.0"
```

Comment:
It would be great if we could use version `4.1.3`, as the current one is vulnerable (https://rustsec.org/advisories/RUSTSEC-2024-0344.html).

Reply:
done